### PR TITLE
[Translate ru] Resources -> markdown syntax

### PR DESCRIPTION
**Pull request by DmitrySkripunov**
[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/ru/review/372)

### CODE REVIEW

**Comment by DmitrySkripunov** [comment]
@petele Hi! I have a strange trouble with Travis here. Please, look at it. Is it problem Travis itself, maybe?
Translate via GitLocalize

**Comment by ilyaspiridonov** [comment]
@agektmr Reviewed

File: src/content/ru/resources/markdown-syntax.md [hunk 1]
```diff
@@ -0,0 +1,617 @@
...
+// ...
+// ...
+</pre>
+```
+
+### Копирование по клику
+
+Копирование по клику автоматически доступно для всех блоков с кодом.
+
+### Особый случай: Шаблоны -  {{}}
```
Comment (agektmr) [inline src/content/ru/resources/markdown-syntax.md]: @ilyaspiridonov If you look at [the original source code](https://raw.githubusercontent.com/google/WebFundamentals/master/src/content/en/resources/markdown-syntax.md), this section is intentionally escaped as `&#123;&#123;}}`, however GitLocalize decodes it unexpectedly. This causes our test runner to return an error. I'll file an issue for this.

Comment (agektmr) [reply src/content/ru/resources/markdown-syntax.md]: https://github.com/gitlocalize/feedback/issues/74

**Pull request merged**